spec/soapy_bing/oauth_credentials_spec.rb

```diff
@@ -79,7 +79,7 @@
 
   describe '#access_token' do
     let(:credentials) { { client_id: 'foo', client_secret: 'bar', refresh_token: 'baz' } }
-    let(:response) { double(:response) }
+    let(:response) { double(:response) } # rubocop:disable RSpec/VerifiedDoubles
```

Comment:
There is [`HTTPParty::method_missing`](https://github.com/jnunemaker/httparty/blob/v0.14.0/lib/httparty/response.rb#L79) which catch `[]` used in tests. So RSpec can not verify it.